packages/pass-style/src/error.js

```diff
@@ -108,18 +107,23 @@ export const confirmRecursivelyPassableErrorPropertyDesc = (
     );
   }
   if (!hasOwn(desc, 'value')) {
-    return (
-      reject &&
-      reject`Passable Error ${q(
-        propName,
-      )} own property must be a data property: ${desc}`
-    );
+    // Without lockdown, Error.prototype.stack is typically an accessor property.
```

Comment:
What do you mean by "typically"? On v8 without lockdown, it is an own accessor property.